setup.cfg

```diff
@@ -45,6 +45,7 @@ docs =
     sphinx-rtd-theme
     sphinxcontrib-contentui
     sphinxcontrib-details-directive
+    nbsphinx
```

Comment:
When I tried to build the docs I also needed to install pandoc, I assume that's related to this? Anyway to document this?

Reply:
I see, thanks for pointing this out. I'll document it in the "development installation" 

I'll check, and I'll also check with Chris whether there are now tools that could avoid the pandoc dependency.

Reply:
@csadorf I now switched to the myst route which should get rid of the pandoc dependency.